initrd/bin/gui-init

```diff
@@ -465,10 +465,12 @@ show_options_menu()
     C )
       luks_reencrypt
       luks_secrets_cleanup
+      prompt_update_checksums
```

Comment:
Is that enough? Afk still.

Reply:
Oh right, I didn't think of that case. I guess there should be a window asking the user whether a TPM DUK should be set up again after passphrase change? Does there need to be a new TPM DUK if only the volume key was changed (I think no, but I'm not sure)?

Reply:
Header will change on either pass change or reenncryption.

gui-init calls this function when tpm is reset or totp is resealed, doing needed magic

https://github.com/linuxboot/heads/blob/6b936e76aa941841148d34b1626c5fd2dd259617/initrd/etc/functions#L83

Reply:
> Header will change on either pass change or reenncryption.

Agreed. But that wasn't my question; I was wondering if a new TPM DUK needs to be generated if the user only does a re-encryption of the LUKS container, without changing his passphrase (I'm thinking no, but wanted confirmation). For the case (or cases) where it does need to be set, the reseal function seems to be the correct one, as it also calls `kexec-seal-key`, which will generate a new TPM DUK, though it wouldn't be the best UX, as that function will ask the user to enter the Disk Recovery Key passphrase, which he had just entered earlier, at the start of the process.

Anyway, the TPM adds quite a bit of complexity to this PR...maybe I should remove that commit and make the PR only about the "clear" command? I can't test any of the code that involves TPM DUK stuff.

Reply:
>>Header will change on either pass change or reenncryption.
>Agreed. But that wasn't my question; I was wondering if a new TPM DUK needs to be generated if the user only does a re-encryption of the LUKS container, without changing his passphrase (I'm thinking no, but wanted confirmation). 

@UndeadDevel : the sealing function takes into consideration hash of luks header, so yes, in both cases TPM DUK needs to be resealed as per PCR6 measurement

https://github.com/linuxboot/heads/blob/6b936e76aa941841148d34b1626c5fd2dd259617/initrd/bin/kexec-seal-key#L134

Created by
https://github.com/linuxboot/heads/blob/6b936e76aa941841148d34b1626c5fd2dd259617/initrd/bin/qubes-measure-luks#L18

and then part of TPM DUK unsealing at 
https://github.com/linuxboot/heads/blob/6b936e76aa941841148d34b1626c5fd2dd259617/initrd/bin/kexec-unseal-key#L36

Called by
https://github.com/linuxboot/heads/blob/6b936e76aa941841148d34b1626c5fd2dd259617/initrd/bin/kexec-insert-key#L46